cmake/common/versionconfig.cmake

```diff
@@ -33,23 +33,22 @@ endif()
 
 # Set beta/rc versions if suffix included in version string
 if(_obs_version MATCHES "[0-9]+\\.[0-9]+\\.[0-9]+-rc[0-9]+")
-  string(REGEX REPLACE "([0-9]+)\\.([0-9]+)\\.([0-9]+)-rc([0-9]+).*$" "\\1;\\2;\\3;\\4" _obs_release_candidate
-                       ${_obs_version})
+  string(REGEX REPLACE "([0-9]+)\\.([0-9]+)\\.([0-9]+)-rc([0-9]+).*$" "\\4" _obs_release_candidate ${_obs_version})
```

Comment:
We match 4 tokens if you need only the last one - I'd just remove the parens around the first 3 tokens and only match the number after `rc` or `beta` respectively.

Reply:
Yeah I just forgot about that.